RevenueCatUI/Templates/OnePackageWithFeaturesTemplate.swift

```diff
@@ -0,0 +1,192 @@
+    private var introEligibility: IntroEligibilityStatus?
+    private var localization: ProcessedLocalizedConfiguration
+
+    @EnvironmentObject
+    private var purchaseHandler: PurchaseHandler
+
+    init(configuration: TemplateViewConfiguration, introEligibility: IntroEligibilityStatus?) {
+        self.configuration = configuration
+        self.introEligibility = introEligibility
+        self.localization = configuration.packages.single.localization
+    }
+
+    var body: some View {
+        ZStack {
+            self.background
+
+            self.content
+        }
+    }
+
+    private var content: some View {
+        VStack {
+            if let url = self.configuration.iconImageURL {
+                RemoteImage(url: url)
+                    .frame(width: self.iconSize, height: self.iconSize)
+                    .cornerRadius(8)
+            }
+
+            Text(self.localization.title)
+                .font(.title)
+                .foregroundStyle(self.configuration.colors.foregroundColor)
+                .multilineTextAlignment(.center)
+
+            Spacer()
+
+            self.features
+                .scrollableIfNecessary()
+
+            Spacer()
+
+            IntroEligibilityStateView(
+                textWithNoIntroOffer: self.localization.offerDetails,
+                textWithIntroOffer: self.localization.offerDetailsWithIntroOffer,
+                introEligibility: self.introEligibility,
+                foregroundColor: self.configuration.colors.accent2Color
+            )
+            .multilineTextAlignment(.center)
+            .font(.subheadline)
+            .padding(.bottom)
+
+            PurchaseButton(
+                package: self.configuration.packages.single.content,
+                purchaseHandler: self.purchaseHandler,
+                colors: self.configuration.colors,
+                localization: self.localization,
+                introEligibility: self.introEligibility,
+                mode: self.configuration.mode
+            )
+            .padding(.bottom)
+
+            FooterView(configuration: self.configuration.configuration,
+                       color: self.configuration.colors.accent2Color,
+                       purchaseHandler: self.purchaseHandler)
+        }
+        .padding(.horizontal)
+        .padding(.top)
+    }
+
+    private var features: some View {
+        VStack(spacing: 20) {
+            ForEach(self.localization.features, id: \.title) { feature in
+                FeatureView(feature: feature, colors: self.configuration.colors)
+            }
+        }
+        .padding(.horizontal)
+    }
+
+    private var background: some View {
+        Rectangle()
+            .foregroundStyle(self.configuration.colors.backgroundColor)
+            .frame(maxWidth: .infinity, maxHeight: .infinity)
+            .edgesIgnoringSafeArea(.all)
+    }
+
+    @ScaledMetric(relativeTo: .title)
+    private var iconSize = 55
```

Comment:
This is how the template can scale up the icon without hardcoding an absolute number 😇

Reply:
TIL